Tests/XCUITests/PerformanceTests.swift

```diff
@@ -169,6 +175,70 @@ class PerformanceTests: BaseTestCase {
         }
     }
 
+    func testPerfBookmarks1startUp() {
+        waitForTabsButton()
+//        app.terminate()
```

Comment:
Nit: debug line not needed

Reply:
whoops, left it in. good catch. removing...